### PR TITLE
fix: update to support upfront commitment

### PR DESCRIPTION
**Pull request by marie-j**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | yes
| New feature?     |no
| Breaking change? |no
| Tickets          |  MANAGER-6143
| License          | BSD 3-Clause

### CODE REVIEW

**Review by JeremyDec — DISMISSED** [review]

**Review by frenautvh — DISMISSED** [review]

**Review by JeremyDec — DISMISSED** [review]

**Comment by marie-j** [comment]
Rebase to fix conflicts

File: packages/manager/modules/billing/package.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
     "angular": "^1.7.5",
     "angular-translate": "^2.18.1",
     "bootstrap": "~3.3.7",
-    "bootstrap4": "twbs/bootstrap#v4.0.0"
+    "bootstrap4": "twbs/bootstrap#v4.0.0",
```
Comment (radireddy) [inline packages/manager/modules/billing/package.json:31]: bootstrap 4 required? if so is it possible to remove bootstrap 3?

Comment (marie-j) [reply packages/manager/modules/billing/package.json:31]: bootstrap4 required for utilities, bootstrap 3 for the grid (still bootstrap3 in Dedicated app)

**Review by JeremyDec — DISMISSED** [review]

**Review by mohammed-zahaf — APPROVED** [review]

**Review by ganeshkumar1989 — APPROVED** [review]

**Comment by marie-j** [comment]
Rebased on base branch

**Pull request merged**